ios/Video/RCTVideoManager.m

```diff
@@ -57,12 +59,17 @@ @interface RCT_EXTERN_MODULE (RCTVideoManager, RCTViewManager)
 RCT_EXPORT_VIEW_PROPERTY(onPlaybackStalled, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onPlaybackResume, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onPlaybackRateChange, RCTDirectEventBlock);
+RCT_EXPORT_VIEW_PROPERTY(onPlayedTracksChange, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onVolumeChange, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onVideoPlaybackStateChanged, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onVideoExternalPlaybackChange, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onGetLicense, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onPictureInPictureStatusChanged, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onRestoreUserInterfaceForPictureInPictureStop, RCTDirectEventBlock);
+
+RCT_EXPORT_VIEW_PROPERTY(onCommandResult, RCTBubblingEventBlock);
+RCT_EXTERN_METHOD(getCurrentTime : (nonnull NSNumber*)rectTag reactTag : (nonnull NSNumber*)commandId commandId)
```

Comment:
Wydaje się że możemy usunąć tą implementacje bo jest zaimplementowane w open-source

https://thewidlarzgroup.github.io/react-native-video/component/methods#getcurrentposition

Musimy tylko zaktualizować main-dxp zmianami które wyszły w 6.2.0 open-source. Zrobimy to po zakończeniu tego wyrównania z 5.X

Reply:
okej, usunę ten commit